### DISCUSSION TITLE
Mouse wheel works for scrolling but not zooming in

### DISCUSSION THREAD

**Discussion by pedrib** [i3/i3#5491] [Q&A]
Hi all,

Something that has been bugging me for a while is that while I can scroll in any application with my mouse wheel, it seems that the zooming function with CTRL + mouse wheel is not working all applications that I have tested (Chrome, Firefox, GIMP, PDF readers). 
I don't have anything tying up the mouse wheel or the CTRL key in my i3 config, any ideas what might be going wrong here, and how I can debug it?

Thanks for the help!

**Comment by orestisfl** [comment]
Ideas:
1. Debug mouse events with xev
2. Share your config and/or i3 log file
3. Check if it works with a different wm like awesome and openbox

**Comment by hoshmandent** [comment]
you can try the following steps to debug the issue:

1. Check if the CTRL key is functioning properly by testing it with other key combinations. For example, try using CTRL + C to copy text, or CTRL + V to paste text.

2. Check if the mouse wheel is functioning properly by testing it with other applications. For example, try using the mouse wheel to scroll up and down in a web browser or text editor.

3. Verify that the i3 configuration is not interfering with the zooming function. Look for any bindings or settings related to the mouse wheel or the CTRL key that could be affecting the zooming function.

4. Check if the zooming function works in other window managers or desktop environments. If it does, then the issue is likely with i3.

5. If none of the above steps work, you can try using xev, a tool that displays X events generated by your mouse and keyboard. You can use xev to verify if the CTRL + mouse wheel event is being recognized by your system. If xev shows that the event is being generated correctly, then the issue is likely with the application itself.

**Comment by pedrib** [comment]
Turns out the problem was between the chair and the keyboard 🗡️ 

I had `imwheel` running and it was taking over the zoom functionality. You can check here for a solution if you're also using imwheel: https://github.com/freeplane/freeplane/issues/134#issuecomment-1566269800

Thank you for your help and sorry for wasting your time.
